packages/apollo-angular/src/query-ref.ts

```diff
@@ -1,31 +1,26 @@
-import { Observable } from 'rxjs';
+import { from, Observable } from 'rxjs';
 import { NgZone } from '@angular/core';
 import type {
-  ApolloQueryResult,
-  FetchMoreQueryOptions,
+  ApolloClient,
   MaybeMasked,
   ObservableQuery,
   OperationVariables,
-  SubscribeToMoreOptions,
   TypedDocumentNode,
-  Unmasked,
-} from '@apollo/client/core';
+} from '@apollo/client';
 import { EmptyObject } from './types';
-import { fromObservableQuery, wrapWithZone } from './utils';
+import { wrapWithZone } from './utils';
 
 export type QueryRefFromDocument<T extends TypedDocumentNode> =
   T extends TypedDocumentNode<infer R, infer V> ? QueryRef<R, V & OperationVariables> : never;
 
 export class QueryRef<TData, TVariables extends OperationVariables = EmptyObject> {
-  public readonly valueChanges: Observable<ApolloQueryResult<TData>>;
-  public readonly queryId: ObservableQuery<TData, TVariables>['queryId'];
```

Comment:
`QueryManager` now keeps track of `ObservableQuery` instances in a `Set` so there is no more `queryId`.

Reply:
LGTM